### PR TITLE
Added some  info on index.html

### PR DESCRIPTION
**Pull request by DasSkelett**
A bit more than Hello World

### CODE REVIEW

**Comment by lukeramsden** [comment]
Maybe a link to the GitHub would be nice? And some styling?

File: server/index.html [hunk 1]
```diff
@@ -1,5 +1,8 @@
 <html>
 <body>
-Hello World!
+Hi, this is chaosbot. <br>
+For more information see <br>
+  <a href="https://github.com/chaosbot/chaos">Github</a> !
```
Comment (mgiuffrida) [inline server/index.html]: remove space before !

File: server/index.html [hunk 2]
```diff
@@ -1,5 +1,8 @@
 <html>
 <body>
-Hello World!
+Hi, this is chaosbot. <br>
```
Comment (mgiuffrida) [inline server/index.html]: ChaosBot, as in README.md.

**Review by mgiuffrida — CHANGES_REQUESTED** [review]

**Review by pchauncey — APPROVED** [review]

**Comment by richardneitzke** [comment]
👍

**Comment by psykzz** [comment]
Conflicts

**Review by davidak — APPROVED** [review]

**Comment by DasSkelett** [comment]
Resolved conflicts.
Please re-vote / re-approve, else your likes don't count.
(I think this isn't fixed yet)

**Review by davidak — APPROVED** [review]

**Pull request merged**